op-bindings/predeploys/addresses_test.go

```diff
@@ -36,5 +36,5 @@ func TestL1BlockSlots(t *testing.T) {
 
 	require.Equal(t, types.OverheadSlot, overHeadSlot)
 	require.Equal(t, types.ScalarSlot, scalarSlot)
-	require.Equal(t, types.L1BaseFeeSlot, l1BaseFeeSlot)
```

Comment:
Should this also include test coverage of the new slot + offsets?

Reply:
yes, added!

Reply:
well added slot tests, not offset. Will have to look into how to check offsets later.

Reply:
oh I was just adding the same test